src/DataManagement/EntryList.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using ePiggyWeb.Utilities;
+
+namespace ePiggyWeb.DataManagement
+{
+    public class EntryList : List<Entry>
+    {
+        //EntryType is to save up on the amount of times we have to pass the EntryType in methods
+        public EntryType EntryType { get; set; }
+
+        public EntryList() { }
+
+        public EntryList(EntryType entryType)
+        {
+            EntryType = entryType;
+        }
+
+        public override string ToString()
+        {
+            var sb = new StringBuilder();
+
+            foreach (var entry in this)
+            {
+                sb.Append(entry);
+                //sb.Append("\n");
+            }
+
+            return sb.ToString();
```

Comment:
Where are you seeing this `ToString()` to be used?

Reply:
This is not a critically important method, but might be useful for testing purposes, also the ToString method in Entry showcases was useful to test out reflection :D